torchtune/models/code_llama2/_model_builders.py

```diff
@@ -29,7 +29,9 @@ def code_llama2_7b() -> TransformerDecoder:
         embed_dim=4096,
         max_seq_len=16384,
         attn_dropout=0.0,
+        intermediate_dim=11008,
```

Comment:
just sanity check: why update only intermediate_dim on this one? Have we successfully loaded the models for all of them?